### PR TITLE
Fixes #3238 - respect --no-colors option

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

**Comment by ares** [comment]
this will have to be rebased after #75

**Comment by ares** [comment]
Rebased

File: config/foreman-installer.yaml [hunk 1]
```diff
@@ -9,6 +9,7 @@
 :modules_dir: .
 
 ## Kafo tuning, customization of core functionality
+:name: Foreman
```
Comment (GregSutcliffe) [inline config/foreman-installer.yaml:12]: Whats this for?

Comment (ekohl) [inline config/foreman-installer.yaml:12]: This change seems unrelated. What's it for?

Comment (ares) [inline config/foreman-installer.yaml:12]: Kafo in some previous version enabled installers to set their name to be displayed in interactive mode. Should I split to a separate PR?

Comment (GregSutcliffe) [inline config/foreman-installer.yaml:12]: I'm fine with it, just wanted to know what it was :)

**Comment by domcleal** [comment]
Merged as 3897a1a, thanks @ares!
